server/views/pages/bookings/visit.njk

```diff
@@ -14,6 +24,35 @@
 
       <p>Booking reference: <span data-test="booking-reference">{{ visit.reference }}</span></p>
 
+      {% if visit.visitStatus === "CANCELLED" %}
+
+        {% set status = visit.outcomeStatus.split("_") %}
+            
+        {% set cancelledBy %}
+          {% if status[0] === 'VISITOR' %}
+            {{- 'This visit was cancelled by a visitor.' -}}
+          {% elif status[0] === 'PRISONER' %}
+            {{- 'This visit was cancelled by the prisoner.' -}}
+          {% else %}
+            {{- 'This visit was cancelled by the prison.' -}}
+          {% endif %}
+        {% endset %}
```

Comment:
Looking at the definition of `outcomeStatus` it's got 20 values and more than 1 would match either `visitor_` or `prisoner_`. Assuming this logic is from the staff service but we might want to check which values **actually** get set and whether this logic is still correct or whether we should be matching specific values.